csu_radartools/common.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+"""
+common sub-module of csu_radartools
+
+Contains commonly used functions.
+
+Nick Guy 10 May 2016
+"""
+import numpy as np
+
+##############
+##  Arrays  ##
+##############
+
+def _check_for_array(dz, zdr, kdp):
+    len_flag = hasattr(dz, '__len__')
+    if not len_flag:
+        dz = np.array([dz])
+        kdp = np.array([kdp])
+        zdr = np.array([zdr])
+    return dz, zdr, kdp, len_flag
+
+########################
+##  Unit Conversions  ##
+########################
+def dbz2z(dbz):
```

Comment:
Two line breaks before a `def` declaration, according to pep8 (a standard `csu_radartools` meets).